src/main/java/uk/gov/hmcts/darts/common/repository/TransformedMediaRepository.java

```diff
@@ -17,6 +18,13 @@ public interface TransformedMediaRepository extends JpaRepository<TransformedMed
         """)
     List<TransformedMediaEntity> findByMediaRequestId(Integer mediaRequestId);
 
+
+    @Query("""
+        SELECT tm FROM TransformedMediaEntity tm
+        WHERE tm.mediaRequest.id IN :ids
+        """)
+    List<TransformedMediaEntity> findByIdIn(List<Integer> ids);
```

Comment:
should probably call this `findByMediaRequestIdIn` or `findByMediaRequestIds`

Reply:
It's not being used so I will delete it.